### PR TITLE
Build cleanup

### PR DESCRIPTION
**Pull request by blink1073**
Models our python build after `nbdime` and adds basic install check on `appveyor`.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Do not build feature branch with open Pull Requests
+skip_branch_with_pr: true
+
+# scripts that are called at very beginning, before repo cloning
+init:
+  - ps: (new-object net.webclient).DownloadFile('https://github.com/git-for-windows/git/releases/download/v2.11.0.windows.1/Git-2.11.0-64-bit.exe', 'C:\\git-setup.exe')
+  - cmd: START /WAIT C:\\git-setup.exe /VERYSILENT
```
Comment (vidartf) [inline appveyor.yml]: Unless you have a specific need for a newer version of git you can remove this bit :)

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  - 'SET "PATH=%PYTHON%\\Scripts;%PATH%"'
+  # Install our package:
+  - 'pip install --upgrade -e ".[test]"'
+
+build: off
+
+# scripts to run before tests
+before_test:
+  - git config --global user.email appveyor@fake.com
+  - git config --global user.name "AppVeyor CI"
```
Comment (vidartf) [inline appveyor.yml]: This is also only relevant if you call out to git during your tests.

**Comment by blink1073** [comment]
Thanks @vidartf, I removed the appveyor file altogether because I couldn't figure out how to get appveyor enabled for this org.

**Comment by blink1073** [comment]
Figured it out.

**Comment by blink1073** [comment]
Ugh, no I didn't...  There is a granularity of permissions problem.

**Comment by blink1073** [comment]
Found it!

**Pull request merged**